torch/_dynamo/variables/builtin.py

```diff
@@ -1164,8 +1164,10 @@ def _grad_changed(old, new):
                 return TorchVariable(member, **options)
             elif ConstantVariable.is_literal(member):
                 return ConstantVariable.create(member, **options)
```

Comment:
NIT: these cases are handled by the `SourcelessBuilder`
```suggestion
```

Reply:
Just the three you linked or some more above that you couldn't link because the GH UI wouldn't let you?

Reply:
Ok, seems to be just these three lines. At least, I can't find any code for the rest in `SourcelessBuilder.__call__`.